src/test/groovy/com/palantir/gradle/docker/DockerRunPluginTests.groovy

```diff
@@ -201,6 +201,53 @@ class DockerRunPluginTests extends AbstractPluginTest {
         buildResult.output =~ /(?m):dockerRunStatus\nDocker container 'foo' is STOPPED./
     }
 
+    def 'can mount volumes specified by a File object with an absolute path'() {
```

Comment:
I would prefer we change this test to just "can mount volumes specified with an absolute path" and use the absolute path string in the build file. While using a File object works, its a bit awkward and I would rather not endorse that usage in the tests.

Reply:
Yep, I've updated the PR. Thanks!